app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @user.update(user_params)
+    redirect_to new_user_path
+  end
```

Comment:
- [ ] Similarly to the `#create` action, if the user is not successfully saved, it should render the `:edit` form again. 
![Screen Shot 2020-11-18 at 6 38 37 PM](https://user-images.githubusercontent.com/48804258/99566498-46ee7800-29cd-11eb-9852-e59a8dd6db4e.png)


- [ ] It should also redirect to the `edit_user_path` if the user was saved.
![Screen Shot 2020-11-18 at 6 34 25 PM](https://user-images.githubusercontent.com/48804258/99566038-b021bb80-29cc-11eb-8a44-36d8a56bc3c0.png)